app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% I18n.t provide(:title, t("static_pages.home.about"))%>
```

Comment:
1. chi ri em, em phải code như ri chứ:
`I18n.t provide(:title, t("static_pages.home.about"))`
thành `provide :title, t("static_pages.home.about")`

2. `The`.... em dùng I18n nhé

3. Sửa các file html bên dưới nhé em